src/assets/sass/protocol/base/elements/_forms.scss

```diff
@@ -128,10 +127,99 @@ fieldset li {
         }
     }
 
-    select,
     input[type='checkbox'],
     input[type='radio'] {
         box-shadow: 0 0 0 2px $color-red-50;
     }
 
 }
+
+
+/* -------------------------------------------------------------------------- */
+// Select
+
+// with thanks to https://www.filamentgroup.com/lab/select-css.html
+select {
+    @include border-box;
+    @include appearance(none);
+    @include form-input;
+    background-image: $url-image-arrow-down-form,
+                      linear-gradient(to bottom, #ffffff 0%,#ffffff 100%); // must keep gradient to avoid bugs
+	background-position: right 8px top 50%;
+    background-repeat: no-repeat, repeat;
+    background-size: 1.5em auto, 100%;
+	display: block;
+	font-weight: normal;
+	max-width: 100%;
+	min-width: $field-min-width;
+    padding: $field-padding calc(1.5em + #{$field-padding * 2}) $field-padding $field-padding;
+    text-overflow: ellipsis;
+
+    [dir="rtl"] & {
+        background-position: left 8px top 50%;
+        padding: $field-padding $field-padding $field-padding calc(1.5em + #{$field-padding * 2});
+    }
+
+    // no down arrow on multi selects
+    &[multiple] {
+        &,
+        &:hover,
+        &:focus {
+            background: #fff;
+            padding: $field-padding;
+        }
+    }
+
+    &::-ms-expand {
+        display: none;
+    }
+
+    &:hover {
+        border-color: $color-link-hover;
+        background-image: $url-image-arrow-down-link-hover,
+	                  linear-gradient(to bottom, #ffffff 0%,#ffffff 100%); // must keep gradient to avoid bugs
```

Comment:
`$color-white`? Or since this gradient seems to be used several times is it worth making a variable for it in the lib?